src/foam/u2/detail/SectionView.js

```diff
@@ -78,6 +78,17 @@ foam.CLASS({
           return this.getNestedPropValue(obj[props.shift()], props.join('.'))
         }
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'loadLatch',
+      preSet: function(o, n) {
+        return this.sectionContentViewSet ? o || n : n;
```

Comment:
Couldn't this just be 'o || n' and then no need for sectionContent since this value will only change once, ie. from false -> true?

Reply:
false will be changed to true if tab isn't selected in TabbedDetailView and we need to ignore it

Reply:
so false is default and on false we load section content as views elsewhere other then TabbedDetailVIew will work with `loadLatch` set to false and we need content of the section loaded

Reply:
false -> true will be changed in src/foam/u2/detail/TabbedDetailView.js line 85 for tabs which are not selected

Reply:
and then when we select it back from true ->  false we aren't loading it again as we have true || false -> true on true we don't load

Reply:
but we cannot load on true value as it's not default and other usages of SectionView will not load the contect